scripts/process_data.py

```diff
@@ -34,6 +37,19 @@ class CameraModel(Enum):
 }
 
 
+def status(msg: str, spinner: str = "bouncingBall", verbose: bool = False):
+    """A context manager that does nothing is verbose is True. Otherwise it hides logs under a message.
+
+    Args:
+        msg: The message to log.
+        spinner: The spinner to use.
+        verbose: If True, print all logs, else hide them.
+    """
+    if verbose:
+        return nullcontext()
```

Comment:
Is this flipped?

Reply:
We don't want to use `CONSOLE.status` when verbose is used, otherwise all of the outputs will be hidden.